examples/tests/crash_immediately_syscall/main.c

```diff
@@ -0,0 +1,29 @@
+// Do nothing. yield() immediately.
+
+// Define the `_start_` signature.
+void _start(void* app_start, void* mem_start, void* memory_len, void* app_heap_break);
+
+__attribute__((naked))
+void _start(void* app_start __attribute__((unused)),
+            void* mem_start __attribute__((unused)),
+            void* memory_len __attribute__((unused)),
+            void* app_heap_break __attribute__((unused))) {
+  // invalid syscall
+#if defined(__thumb__)
+  __asm__ volatile (
+    "svc 0xbd       \n"
+    :
+    :
+    : "memory", "r0", "r1", "r2", "r3", "r12", "lr"
+    );
+#elif defined(__riscv)
+  __asm__ volatile (
+    "li    a4, 0xbd\n"
```

Comment:
```suggestion
    "li    a0, 0xbd\n"
```
?

Reply:
`a4` is the syscall argument register on RISC-V now: https://github.com/tock/tock/blob/master/doc/reference/trd104-syscalls.md#31-registers